cli/src/cpp/utils/participant_subelement_parser.cpp

```diff
@@ -176,7 +176,47 @@ void participant_subelement_parser(
     }
     else if (element == NAME_SUBELEMENT)
     {
-        std::cout << "Participant name configuration not yet supported" << std::endl;
+        // Check help value
+        print_usage = check_help(values);
+        // Not keyed element
+        print_usage = print_usage || !check_keyed(false, keyed, message.str());
+        // Final element
+        print_usage = print_usage || !check_final_element(true, subelement, message.str());
+        // SET command requires one argument
+        print_usage = print_usage || (CommonCommands::SET == command && !check_command_arguments(command, 1,
+                values.size(), message.str(), true));
+        // PRINT and CLEAR require no argument
+        print_usage = print_usage || ((CommonCommands::CLEAR == command || CommonCommands::PRINT == command) &&
+                check_command_arguments(command, 0, values.size(), message.str(), true));
+        // Query command is not allowed: not collection element.
+        print_usage = print_usage || (CommonCommands::QUERY == command && query_not_allowed(message.str()));
+
+        if (!print_usage)
+        {
+            try
+            {
+                switch (command)
+                {
+                    case CommonCommands::CLEAR:
+                        qosprof::domain_participant::clear_name(filename, profile_name);
+                        break;
+                    case CommonCommands::PRINT:
+                        qosprof::domain_participant::print_name(filename, profile_name);
+                        break;
+                    case CommonCommands::SET:
+                        qosprof::domain_participant::set_name(filename, profile_name, values[0]);
+                        break;
+                }
+            }
```

Comment:
Leveraging the switch, maybe we could move the checks inside it ? What do you think
Something like the following 

```suggestion
        if (!print_usage)
        {
            try
            {
                switch (command)
                {
                    case CommonCommands::CLEAR:
                    {
                         print_usage = !check_command_arguments(command, 0, values.size(), message.str(), true);
                         if (!print_usage)
                         {
                             qosprof::domain_participant::clear_name(filename, profile_name);
                         }
                         break;
                    }
                    case CommonCommands::PRINT:
                    {
                         print_usage = !check_command_arguments(command, 0, values.size(), message.str(), true);
                         if (!print_usage)
                         {
                             qosprof::domain_participant::print_name(filename, profile_name);
                         }
                         break;
                    }
                    case CommonCommands::QUERY:
                     {
                         print_usage = query_not_allowed(message.str());
                         break;
                    }
                    case CommonCommands::SET:
                    {
                         print_usage = !check_command_arguments(command, 1,
                values.size(), message.str(), true);
                         if (!print_usage)
                         {
                             qosprof::domain_participant::set_name(filename, profile_name, values[0]);
                         }
                         break;
                    }
                }
            }
```